src/main/resources/application.yml

```diff
@@ -11,7 +11,7 @@ spring:
   jpa:
     database-platform: org.hibernate.dialect.PostgreSQL94Dialect
     hibernate:
-      ddl-auto: update
+      ddl-auto: create-drop
```

Comment:
This should be changed back - it'll be annoying for people to keep losing the data in the DB